src/parsers/manifest/dash/node_parsers/AdaptationSet.ts

```diff
@@ -123,7 +123,11 @@ function parseAdaptationSetChildren(
       switch (currentElement.nodeName) {
 
         case "Accessibility":
-          children.accessibility = parseScheme(currentElement);
+          if (children.accessibilities == null) {
```

Comment:
We now prefer to always use strict equality (`===`) to be more explicit about which values are tested (so here we just check for undefined, not `null` or `undefined`).

There are examples in that file where we don't, but that's only because it is older code.
We plan to replace all of them at some point. Once this will be done, we will be able to add this rule to our linter, so devs would be alerted much sooner of this.